index.bs

```diff
@@ -7878,6 +8674,45 @@ partial dictionary MLOpSupportLimits {
   </details>
 </div>
 
+### tile ### {#api-mlgraphbuilder-tile}
+Repeat a tensor the number of times along each dimension.
+
+<script type=idl>
+partial interface MLGraphBuilder {
+  MLOperand tile(MLOperand input,
+                 sequence<unsigned long> repetitions,
+                 optional MLOperatorOptions options = {});
+};
+
+partial dictionary MLOpSupportLimits {
+  MLSingleInputSupportLimits tile;
+};
+</script>
+
+<div dfn-for="MLGraphBuilder/tile(input, options)" dfn-type=argument>
+    **Arguments:**
+        - <dfn>input</dfn>: an {{MLOperand}}. The input N-D tensor.
+        - <dfn>repetitions</dfn>: A count per each dimension of how many times to repeat that dimension. The repetitions count must match the input rank, using 1's for any axis that should retain the same size.
```

Comment:
Wording: "per each" seems redundant. Either "count per dimension" or "count for each dimension" ?